lib/systems/platforms.nix

```diff
@@ -521,5 +521,7 @@ rec {
 
     else if platform.parsed.cpu == lib.systems.parse.cpuTypes.powerpc64le then powernv
 
-    else pc;
+    else if platform.isAvr then pc   # not sure this is right, but preserves existing behavior
```

Comment:
AVR is a tiny 8 bit arch. You can just drop this.

Reply:
Done.